README.md

```diff
@@ -1,4 +1,52 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql/README.md)
-2. [Core Java Apps](./core_java/README.md)
-3. [Springboot Trading REST API](./springboot/README.md) In-progress
+# Anne Nguyen . Jarvis Consulting
+
+Prior to pursuing a career in data science, I completed my first undergraduate degree at Waterloo in Health Science, where I picked up my interest in technology through participating in the VeloCity residency startup initiative. From this interest, I pursued a B.Sc. in computer science at Ryerson University, which I completed recently. In my spare time, I write and publish creative writing online, go to board game cafés with my friends, and watch and listen to a significant amount of baseball. During a machine learning course, I took the opportunity to analyze a decade's worth of baseball statistics to analyze individual player performance - information I knew as a fan reading FanGraph analysis, but never computed or compiled myself. Through this course, I became more earnestly interested in data science.
+
+## Skills
+
+**Proficient:** Java, Bash, SQL, Agile/Scrum
+
+**Competent:**  C/C++, NumPy, pandas, Python, Spring, Spring Boot.
+
+**Familiar:** HTML/CSS, PHP, Javascript. 
+
+## Development Projects
+
+[Project source code.](https://github.com/jarviscanada/jarvis_data_eng_anne)
+
+- **[Cluster Monitor Agent](./linux_sql)**: An internal tool that monitors the server cluster resources. It records the hardware specifications of each node (server) and monitors resource usage such as CPU and memory utilization in real time. The collected data is stored in a locally hosted PostgreSQL database on a designated primary server node. This tool allows for report generation, which can inform future resource planning such as adding or removing servers based on their usage and activity.
```

Comment:
Please run a spelling and grammar check using https://www.grammarly.com/ if necessary